tests/scripts/all.sh

```diff
@@ -2105,6 +2105,7 @@ component_test_psa_crypto_config_accel_hash_use_psa () {
     scripts/config.py unset MBEDTLS_HKDF_C # has independent PSA implementation
     scripts/config.py unset MBEDTLS_HMAC_DRBG_C
     scripts/config.py unset MBEDTLS_ECDSA_DETERMINISTIC
+    scripts/config.py unset MBEDTLS_SSL_EARLY_DATA
```

Comment:
Without MBEDTLS_SSL_EARLY_DATA disabled in this case, I can run it successfully locally, but in one time CI failure,  I just got 0 result and didn't get the running log. So I unset the MBEDTLS_SSL_EARLY_DATA and the issue gone.   By the way, if any side effect if we add it here?

Reply:
It might hidden real failure.  

With `MBEDTLS_SSL_EARLY_DATA` , it should be pass. If no, we should dig the reason.
Please remove it and check if we can reproduce the failure. 

If it is not relative `EARLY_DATA`, we should report it.

Reply:
> It might hidden real failure.
> 
> With `MBEDTLS_SSL_EARLY_DATA` , it should be pass. If no, we should dig the reason. Please remove it and check if we can reproduce the failure.
> 
> If it is not relative `EARLY_DATA`, we should report it.

Make sense, will change it back and see what happen.